common/build/eslint-plugin-fluid/src/test/enforce-no-unchecked-array-access/enforce-no-unchecked-array-access.test.js

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const assert = require("assert");
+const path = require("path");
+const { ESLint } = require("eslint");
+
+describe("ESLint Rule Tests", function () {
+	function createESLintInstance(config) {
+		return new ESLint({
+			useEslintrc: false,
+			overrideConfig: config,
+			rulePaths: [path.join(__dirname, "../../rules")],
+		});
+	}
+
+	it("Should report an error for unchecked record access but not array access", async function () {
+		const eslint = createESLintInstance({
+			rules: {
+				"no-unchecked-array-access": "error",
+			},
+			parser: "@typescript-eslint/parser",
+			parserOptions: {
+				project: path.join(__dirname, "../tsconfig.json"),
+			},
+		});
```

Comment:
This is going to be exactly the same for each test case, right? So, it should be in the common code. We don't want to accidentally change one case and not the other to get different results.

Reply:
yea forsure, i didnt see it when i copied it over, thank you